webpack.config.js

```diff
@@ -202,7 +202,7 @@ module.exports = [
  * @returns {string|undefined} If the request is to be served by the proxy
  * target, undefined; otherwise, the path to the local file to be served.
  */
-function devServerProxyBypass({ path }) {
+function devServerProxyBypass(path) {
```

Comment:
The change is unacceptable because it's not switching between equivalents! If I had to guess, I'd think that you're using a node version which does not support object destructuring. If you'd like to support such environments, you'd have to change `{ path }` to `request` and then inside the function you'd do `const path = request.path;` But (1) you really shouldn't be using a node version which doesn't support object destructuring and (2) you haven't provided any reasoning so I don't see the point in the backwards compatibility.